components/console/introduction.rst

```diff
@@ -110,6 +110,31 @@ This prints::
 
     HELLO FABIEN
 
+Command Lifecycle
+~~~~~~~~~~~~~~~~~
+
+Commands have three lifecycle methods:
+
+:method:`Symfony\\Component\\Console\\Command\\Command::initialize`
+
+    This method is executed before the ``interact()`` and the ``execute()``
+    methods. It's main purpose is to initialize the variables used in the
+    rest of the command methods.
+
+:method:`Symfony\\Component\\Console\\Command\\Command::interact`
+
+    This method is executed after ``initialize()`` and before ``execute()``.
+    Its purpose is to check if some of the options/arguments are missing
+    and interactively ask the user for those values.
+
+:method:`Symfony\\Component\\Console\\Command\\Command::execute`
+
+    This method is executed after ``interact()`` and ``initialize()``. It
+    usually contains the logic to execute to complete this command task.
```

Comment:
Ok of course,
but ``It usually contains the logic to execute this command task.`` seems not good
What about ``It contains the logic you want the command executes`` ?